src/chrome/content/rules/Schweikert.ch.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	SSL error:
+		- gore
+		- ns1
```

Comment:
`ns1` seems to be gone from DNS, would you update the comment? Otherwise, this PR LGTM

Reply:
Done